### ISSUE TITLE
请问运行结果准确率怎么看？

### ISSUE THREAD

**Issue by yxj1992** [jimenbian/sentiment-analysis#1]
`Run id: X2GQ7H
Log directory: /tmp/tflearn_logs/
---------------------------------
Training samples: 25
Validation samples: 17
--
Training Step: 1  | time: 4.402s
| Adam | epoch: 001 | loss: 0.00000 - acc: 0.0000 | val_loss: 0.69363 - val_acc: 0.4118 -- iter: 25/25
--
[[0.49861583 0.50138414]
 [0.49942857 0.50057137]
 [0.4990096  0.50099045]
 [0.49668297 0.503317  ]
 [0.5001381  0.4998619 ]
 [0.49681374 0.50318635]
 [0.49952838 0.5004716 ]
 [0.4993792  0.50062084]
 [0.49945635 0.5005436 ]
 [0.49686605 0.50313395]
 [0.49935415 0.5006458 ]
 [0.49935263 0.5006474 ]
 [0.49893185 0.5010681 ]
 [0.49781716 0.5021828 ]
 [0.49845394 0.5015461 ]
 [0.49686605 0.503134  ]
 [0.4986591  0.50134087]
 [0.49832577 0.5016743 ]
 [0.49850422 0.5014958 ]
 [0.49901885 0.50098115]
 [0.49669054 0.5033094 ]
 [0.49686566 0.5031344 ]
 [0.4995494  0.5004506 ]
 [0.49870756 0.5012924 ]
 [0.4988313  0.5011687 ]]
`

**Comment by jimenbian** [comment]
不是两列嘛，每一行对应一个句子，左边是0的权重，右边是1的权重。0是负面情绪，1是正面。

**Comment by yxj1992** [comment]
但是不知道每一行这个句子本来是正面还是负面，也就是说没有一个总的准确率。希望您能加上，这样实验结果更完善一些
